packages/material-ui-lab/src/useAutocomplete/useAutocomplete.d.ts

```diff
@@ -35,6 +35,15 @@ export interface UseAutocompleteProps {
    * a different option or changes the character string in the input.
    */
   autoSelect?: boolean;
+  /**
+   * Control if the input should be blurred when an option is selected:
+   *
+   * - `false` the input is not blurred.
+   * - `true` the input is always blurred.
+   * - `touch` the input is blurred only on touch devices.
+   * - `pointer` the input is blurred only on pointing devices.
```

Comment:
I'm making a change to `"mouse"` to improve the terminology, based on https://mobiforge.com/design-development/html5-pointer-events-api-combining-touch-mouse-and-pen.

Reply:
I have also renamed the wording as you can trigger a touch and mouse event on the same device, e.g. Microsoft Surface Pro.